sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -2278,8 +2278,8 @@ def expand(self, pcoll):
 
     if (isinstance(self.table_reference, TableReference) and
         self.table_reference.projectId is None):
-      self.table_reference.projectId = pcoll.pipeline.options.view_as(
-          GoogleCloudOptions).project
+      self.table_reference.projectId = self._project or \
```

Comment:
Because this overrides the existing default, we should add a comment to that effect if someone passes the project flag in

Reply:
Done